h/static/styles/annotator/inject.scss

```diff
@@ -34,28 +34,45 @@ $base-font-size: 14px;
     width: 6px;
   }
 
-  button {
-    @include box-shadow(none);
-    display: inline-block;
-    font-family: h;
-    border: none;
-    cursor: pointer;
-    height: 100%;
-    width: 35px;
-    font-size: 18px;
-    margin: 0;
-    padding: 0;
-    text-align: center;
-    background: $white !important;
-    color: $gray-light !important;
+  &:hover {
+    .annotator-adder-action button {
+      color: $gray-light !important;
+    }
+  }
 
-    &:active {
-      transition: background-color .25s;
-      background-color: $gray-light !important;
+  .annotator-adder-action {
+    float: left;
+
+    button {
+      @include box-shadow(none);
+      display: inline-block;
+      font-family: h;
+      border: none;
+      cursor: pointer;
+      height: 100%;
+      width: 45px;
+      font-size: 18px;
+      margin: 2px 0px;
+      padding: 0;
+      text-align: center;
+      background: $white !important;
+      color: $gray-dark !important;
+    }
+
+    .adder-label {
```

Comment:
Try to keep nesting in CSS to a minimum - as this makes maintenance easier. If you can define a class at the top-level (ie. so you get just `.class` as the selector rather than `.parent .child`) that is preferable.

For newly written styling, we try to use [BEM](https://css-tricks.com/bem-101/) conventions. Legacy CSS in our codebase however isn't written this way and over-uses nesting.